<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WgShortcutKey</class>
 <widget class="QWidget" name="WgShortcutKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>31</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="pb_key2">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>5</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: #f2f0e3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_add1">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>4</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0); 
font: 16pt &quot;黑体&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_add2">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>4</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,0); 
font: 16pt &quot;黑体&quot;;
</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_key1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>5</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: #f2f0e3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_key3">
    <property name="geometry">
     <rect>
      <x>258</x>
      <y>5</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color: #f2f0e3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
